build.sbt

```diff
@@ -12,19 +12,20 @@ scalafmtOnCompile in Compile := true
 
 resolvers += "Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven"
 
+organization := "com.github.mrpowers"
 name := "spark-fast-tests"
 spName := "MrPowers/spark-fast-tests"
 
 spShortDescription := "Fast tests with Spark"
 spDescription := "Test your code quickly"
 
-version := "0.17.0"
+version := "0.17.1"
+crossScalaVersions := Seq("2.11.12", "2.12.7")
```

Comment:
This is my first contribution to a cross build (Scala 2.11.X and 2.12.X for the JVM, not even considering Scala.js, which might not be important for this project).

In the future, we might consider enhance the build to adopt best practices for SBT 1.2.6+

Reply:
Yea, if we can get SBT 1.2.6 to work with Scala 2.11, I'm all for adopting the new SBT best practices